test/BF16/mlp-all-bf16-tpprun.mlir

```diff
@@ -0,0 +1,54 @@
+// RUN: tpp-opt %s -convert-check-to-loops -linalg-ext-to-loops -convert-linalg-to-loops -convert-tpp-to-xsmm -convert-xsmm-to-func -arith-expand -convert-math-to-llvm  -convert-vector-to-scf -convert-scf-to-cf -lower-affine  -reconcile-unrealized-casts |\
```

Comment:
This test could be a benchmark (in `test/Benchmark`) and we should add a C/TPP version of it later, then you don't need to name it "tpprun".

Reply:
I'll move it to test/Benchmark in a different patch. Right now, we're categorizing all BF16 tests into one directory.

Reply:
Good point!